configure.ac

```diff
@@ -3,7 +3,7 @@ dnl Copyright (C) 2011 Red Hat, Inc.
 dnl See COPYING.LIB for the License of this software
 
 AC_INIT(
-    [libstoragemgmt], [1.2.3], [libstoragemgmt-devel@lists.fedorahosted.org],
+    [libstoragemgmt], [1.3.0], [libstoragemgmt-devel@lists.fedorahosted.org],
```

Comment:
Ah, now I understand what you wanted me to update. For some reason I was reading the version number as being correct already, sorry!